pkg/network/ovn_kubernetes.go

```diff
@@ -160,16 +174,42 @@ func boostrapOVN(kubeClient client.Client) (*bootstrap.BootstrapResult, error) {
 		return nil, fmt.Errorf("unable to bootstrap OVN, no master nodes found")
 	}
 
-	ovnMasterNodes := []string{}
+	var ovnMasters ovnMasterSlice
+
 	for _, masterNode := range masterNodeList.Items {
-		ovnMasterNodes = append(ovnMasterNodes, masterNode.Name)
+		var ip string
+		for _, address := range masterNode.Status.Addresses {
+			if address.Type == corev1.NodeInternalIP {
+				ip = address.Address
+				break
+			}
+		}
+		if ip == "" {
+			return nil, fmt.Errorf("No InternalIP found on master node '%s'", masterNode.Name)
+		}
+		parsedIP := net.ParseIP(ip)
+		if parsedIP == nil {
+			return nil, fmt.Errorf("Failed to parse InternalIP '%s' found on master node '%s'", ip, masterNode.Name)
+		}
+		if parsedIP.To4() == nil {
+			// IPv6, wrap address in brackets
+			ip = fmt.Sprintf("[%s]", ip)
+		}
+		ovnMasters = append(ovnMasters, &ovnMaster{Name: masterNode.Name, IP: ip})
 	}
 
-	sort.Strings(ovnMasterNodes)
+	sort.Sort(ovnMasters)
+	ovnMasterNodes := []string{}
+	ovnMasterNodeIPs := []string{}
+	for _, m := range ovnMasters {
+		ovnMasterNodes = append(ovnMasterNodes, m.Name)
+		ovnMasterNodeIPs = append(ovnMasterNodeIPs, m.IP)
+	}
```

Comment:
you could just change `OVNBootstrapResult` to have the array of structs so you don't need to do this

Reply:
True - but the consumer of OVNBootstrapResult needs 2 arrays of strings, anyway, so something similar would be done there instead.

Reply:
Actually, if you use the downward API, I think you can get rid of `{{.OVN_NODES}}` entirely anyways.